src/lib/Microsoft.Fx.Portability.MetadataReader/MemberDependency.cs

```diff
@@ -106,7 +106,7 @@ public override int GetHashCode()
         {
             if (!_hashComputed)
             {
-                _hashCode = ((DefinedInAssemblyIdentity?.ToString() ?? string.Empty) + (MemberDocId ?? string.Empty) + IsPrimitive.ToString()).GetHashCode() ^ CallingAssembly.GetHashCode();
+                _hashCode = ((DefinedInAssemblyIdentity?.ToString() ?? string.Empty) + (MemberDocId ?? string.Empty) + IsPrimitive.ToString((IFormatProvider)CultureInfo.InvariantCulture)).GetHashCode() ^ CallingAssembly.GetHashCode();
```

Comment:
Why are we ToString-ing something that appears to bee a bool?

Reply:
Not sure why. The only thing that I can think of may be hashcodes of string "true" vs "false" is more specific than bool values (less chance of ambiguity).